### PR TITLE
PPP and UG

### PR DESCRIPTION
**Pull request by Evande1**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -238,39 +238,51 @@ Examples:
...
 * `attendance mark 1 l/1 m/0`
 
-### Deleting attendance of student: `attendance delete`
+Expected outcome:
+![AttendanceMark](images/AttendanceMark.png)
+After using ```attendance mark``` command
+
+### Deleting attendance of a student: `attendance delete`
 
-Deletes entire attendance list of student in class list
```
Comment (zzthian) [inline docs/UserGuide.md:266]: good change of language for more clarity

File: docs/UserGuide.md [hunk 2]
```diff
@@ -238,39 +238,51 @@ Examples:
...
+Currently, we support a maximum of 12 lessons. If the size of the attendance list keyed in is 0, the attendance list will be listed as N.A. 
 There can only be a maximum of one attendance list for each student for ease of typing.
 </div>
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If you add an attendance list to a student with an attendance list, it overwrites the current attendance list data.
+Be careful when adding attendance to a student as adding an attendance list to a student with a pre-existing attendance list will overwrite the current attendance list of the student.
 </div>
 
+Expected outcome:
```
Comment (zzthian) [inline docs/UserGuide.md:249]: Good use of images helping to guide users 👍

**Review by gowribhat — APPROVED** [review]
LGTM

**Review by zzthian — APPROVED** [review]
Looks good.

**Pull request merged**